openlibrary/core/lending.py

```diff
@@ -99,7 +99,7 @@ def setup(config):
     try:
         amazon_api = AmazonAPI(
             config_amz_api.key, config_amz_api.secret,
```

Comment:
Can we move this out of lending and into vendors?

Reply:
I tried this, the hooks don't work the same way in vendors v. lending and I don't want to play with this right now.